client/consensus/Cargo.toml

```diff
@@ -17,7 +17,7 @@ sc-client-api = { version = "3.0.0", git = "https://github.com/paritytech/substr
 sp-block-builder = { version = "3.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
 sp-inherents = { version = "3.0.0", git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
 fp-consensus = { version = "1.0.0", path = "../../primitives/consensus" }
-fp-rpc = { version = "1.0.0", path = "../../primitives/rpc" }
+fp-rpc = { version = "1.0.1-dev", path = "../../primitives/rpc" }
```

Comment:
All bumps should be recursive so `fc-consensus` needs to be bumped!

Reply:
Done in 565ea71. And I think that should cover it: every crate that was modified got a bump with `-dev` unless it was already at `-dev`. The exception is the node template and runtime, which appear to not be versioned. Hope I got it right this time.